style.css

```diff
@@ -193,3 +204,83 @@ body {
 }
 
 .lang img { margin-right: 16px; }
+
+.contact-section-bg { background: #fff; }
+
+.contact-section .title {
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 12px;
+}
+
+.contact-section .details {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin-bottom: 21px;
+}
+
+.contact-section {
+  background: #6070ff url("/assets/contact.png");
+  background-position: right;
+  background-size: 50% 90%;
+  background-repeat: no-repeat;
+  padding: 103px 24px 45px 24px;
+  border-top-left-radius: 150px;
+}
```

Comment:
- [ ] Please make sure you use `flexbox` to align all contact section components.
![image](https://user-images.githubusercontent.com/68796232/159533090-598474e7-6237-4b66-a0f2-a957caceab52.png).

- [ ] Kindly add `./` before your URL path to make the `background-image` work online it may work locally but online it's not working.
Make it like that `background: #6070ff url("./assets/contact.png")`. 
- [ ] [Optional] And add `./` for all image source paths in other sections too.

Reply:
thanks 😄

Reply:
Thanks 😄